sound/soc/sof/topology.c

```diff
@@ -2645,11 +2649,5 @@ void snd_sof_free_topology(struct snd_sof_dev *sdev)
 		kfree(sroute->private);
```

Comment:
I am also thinking should we add ```list_del(&sroute->list);``` here. We don't need to keep the list after it is freed